crates/ruff/src/rules/flake8_logging_format/rules.rs

```diff
@@ -146,13 +163,17 @@ pub fn logging_call(checker: &mut Checker, func: &Expr, args: &[Expr], keywords:
             );
 
             // G001 - G004
-            if let Some(format_arg) = call_args.argument("msg", 0) {
+            let msg_pos = usize::from(matches!(logging_call_type, LoggingCallType::LogCall));
+            if let Some(format_arg) = call_args.argument("msg", msg_pos) {
                 check_msg(checker, format_arg);
             }
 
             // G010
             if checker.settings.rules.enabled(Rule::LoggingWarn)
-                && matches!(logging_level, LoggingLevel::Warn)
+                && matches!(
+                    logging_call_type,
+                    LoggingCallType::LevelCall(LoggingLevel::Warn)
```

Comment:
I wonder if we should extend this to warn on `logging.log(logging.WARN, "...")`

Reply:
(Can be in a separate PR if needed, not even sure whether that's correct to flag.)

Reply:
Regarding the level variable, I'm not sure as the docs doesn't mention anything about it although the `WARN` level isn't mentioned in the [levels](https://docs.python.org/3/library/logging.html#logging-levels) table.

If we do decide to change it, it'll need to be updated wherever the `level` argument is accepted.